docs/source/contributors.rst

```diff
@@ -0,0 +1,20 @@
+Contributions Welcome!
+======================
+
+**Contributions**
+
+We will support the efforts within the Hyperledger and Decentralized Identity Foundation (DIF) communities to help create innovative distributed ledger technologies and digital identity networks. We believe that creating cross-project and cross-network standardization and interop along with common development frameworks represents the next step in transforming the digital identity space. In particular we support, and look forward to contributing to, the efforts of Hyperledger Fabric, Indy and the newly proposed Aries project along with DIF’s Identity Hub efforts.
+
+Contributions will be directed towards the community projects above as well as open project extensions within the TrustBloc initiative.
+
+In particular, we will contribute:
+
+Hyperledger Fabric performance and customization improvements. We are excited to contribute improvements we learned while building and taking our network operational.
+Enable DIDs to be managed and exposed from Fabric, and more generically enabling document provenance.
```

Comment:
Enable DIDs to be managed -> Enabling DIDs to be managed
more generically enabling -> more generally enabling?